packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -28,6 +28,62 @@ module.exports = ({ types: t }) => {
   const or = (a, b) => t.logicalExpression("||", a, b);
   const and = (a, b) => t.logicalExpression("&&", a, b);
 
+  const operators = new Set([
+    '+', '-', '*', '%',
+    '<<', '>>', '>>>',
+    '&', '|', '^', '/',
+    '**'
+  ]);
+
+  const updateOperators = new Set([
+    '+', '-'
+  ]);
+
+  function isEqual(arr1, arr2) {
+    return arr1.every((value, index) => {
+      return value.toString() === arr2[index].toString();
+    });
+  }
+
+  function getName(node) {
+    if (node.type === 'ThisExpression') {
```

Comment:
I don't really care; I would just want us to be consistent in the entire codebase. Do we want abstractions over performance or vice versa? Ditto with `.get(...)` vs node access. Let's chose one and stick to that?